src/Protocol/Protocol_1_10.cpp

```diff
@@ -1003,5 +1003,7 @@ void cProtocol_1_10_0::WriteMobMetadata(cPacketizer & a_Pkt, const cMonster & a_
 			a_Pkt.WriteBool(ZombiePigman.IsBaby());
 			break;
 		}  // case mtZombiePigman
+
+		default: break;
```

Comment:
Again, is default appropriate here?

Reply:
I've made everything explicit in the 1.12 protocol but I'm not sure if it makes sense for anything other than the latest protocol.

Reply:
I suppoe this makes sense, although I'd like another dev to weigh in here.

Reply:
If you explicitly listed parrots (added in 1.12), to this (1.10) switch, it's unclear what it could do other than "break". Also every time a mob is added then every protocol needs to be updated whether the protocol supports it or not. The problem being if in 1.13 somebody implements parrots but forgets to add them to the 1.12 protocol, but I think that's less of a problem (and if it does happen, the bug report will have a quick and easy fix).

Reply:
I think leaving out the mobs that the protocol doesn't know is about the best thing we could do.